evap/results/templates/results_semester_detail.html

```diff
@@ -59,7 +59,7 @@ <h3>{{ semester.name }}</h3>
                                             {{ contributor.full_name }}{% if not forloop.last %}, {% endif %}
                                         {% endfor %}
                                     </td>
-                                    {% if course|can_user_see_results:request.user and course.avg_grade %}
+                                    {% if course|can_user_see_grades:request.user and course.avg_grade %}
```

Comment:
the `can_user_see_grades` check (and thus the whole filter) shouldn't be necessary here anymore because the grade is set to None in the view, right?

Reply:
yes, but i had to introduce it in the one commit to move that part around as requested - it is removed some commits later.

Reply:
i don't quite understand, i meant that in this particular commit, you are setting the grades to None if `can_user_see_grades` in the view, but here you are still checking `can_user_see_grades`, which is redundant in my understanding.